packages/merchant-experience-api/src/api/functional.spec.ts

```diff
@@ -0,0 +1,29 @@
+import jestOpenAPI from 'jest-openapi'
+import axios from 'axios'
+const path = require('path')
+
+// Sets the location of your OpenAPI Specification file
+jestOpenAPI(path.join(__dirname, '../openapi.yaml'))
+const merchantBaseUrl = 'http://localhost:8086/v1/merchant'
+describe('router.js', () => {
+  if (process.env.IS_LOCAL_DEVELOPMENT != 'true') {
+    /* eslint-disable */
```

Comment:
This should probably not be here?

Reply:
This was a must, because jest does not allow conditional testing, but with this disable jest tries to run this test.

Eslint disables are allowed in tests if needed, but not in production code. This allow us to create functional tests for local development in experience side.